### PR TITLE
[java] Misc improvements

### LINKED ISSUE #1078: [Java] Handle commands expected by eclipse.java.lsp
The Java language server seems to expect a couple of commands to be available on the client.
For example, on first start when the classpath is incomplete, we see a message with three options.

screen shot 2018-01-18 at 14 30 02

Clicking e.g. on the first results in the following error message:
```
The command 'java.ignoreIncompleteClasspath.help' cannot be executed. There are no active handlers available for the command.
```

### PR DESCRIPTION
**Pull request by AlexTugarev**
fixes #1078

how to test:
- just create a java file not included in a proper java project and open it 
- when you try to call any language feature e.g. content assist, the LS should warn you about an incomplete classpath
- you should be able to ignore it, which has to set the severity of this message to ignore in settings
- reload the browser to verify, that the message is not shown on restart of the LS

### CODE REVIEW

**Comment by akosyakov** [comment]
@AlexTugarev is there a corresponding issue?

**Comment by AlexTugarev** [comment]
@akosyakov, yes, it's https://github.com/theia-ide/theia/issues/1078

File: packages/java/src/browser/java-commands.ts [hunk 1]
```diff
@@ -107,6 +122,18 @@ export class JavaCommandContribution implements CommandContribution, MenuContrib
             },
             isEnabled: () => this.javaClientContribution.running
         });
+        commands.registerCommand(JAVA_IGNORE_INCOMPLETE_CLASSPATH, {
+            execute: async () => {
+                await this.preferencesService.set('java.errors.incompleteClasspath.severity', 'ignore');
```
Comment (akosyakov) [inline packages/java/src/browser/java-commands.ts]: It sets as a user setting. Is it expected?

Comment (akosyakov) [reply packages/java/src/browser/java-commands.ts]: Looks like expected: https://github.com/redhat-developer/vscode-java/blob/b6b922cedc2d0c7e859aedb4792427e9b4ba52a4/src/extension.ts#L393

**Pull request merged**